js/main.js

```diff
@@ -0,0 +1,124 @@
+'use strict';
+var NUMBER_ADDS = 8;
+var LOCATION_Y_MIN = 130;
+var LOCATION_Y_MAX = 630;
+var HEADINGS_ADD = ['Кузькин дом', 'Холупа', 'дом бойца', 'кошкин дом', 'дом у моря', 'самый чистый дом', 'эконном', 'вип дом'];
+var TYPES_ADD = ['Квартира', 'Дом', 'Бунгало', 'Дворец'];
+var PRICES_ADD = [100, 200, 300, 400, 500, 600, 700, 800];
+var ROOMS_ADD = [1, 2, 3, 4];
+var GUESTS_ADD = [1, 2, 3, 4, 5, 6, 7, 8];
+var CHECKINS_ADD = ['12:00', '13:00', '14:00'];
+var DESCRIPTIONS_ADD = ['выбирай нас', 'мы лучшие', 'у нас всегда чисто', 'у нас уютно', 'у нас хорошо', 'к нам студенты', 'к нам вип гости', 'к нам если все занято'];
+var fragment = document.createDocumentFragment();
+var adUnit = document.querySelector('#pin').content.querySelector('.map__pin');
+var adUnitImg = adUnit.querySelector('img');
+var insertionUnit = document.querySelector('.map__pins');
+var mapBlock = document.querySelector('.map');
+var cardAdd = document.querySelector('#card').content.querySelector('.map__card');
+var infoAdds = [];
+
+var cardAddCopy = cardAdd.cloneNode(true);
+var mapSection = document.querySelector('.map');
+var mapSectionChilder = document.querySelector('.map__filters-container');
```

Comment:
```suggestion
var mapSectionChildren = document.querySelector('.map__filters-container');
```

Ты тут имел в виду потомка `mapSection`?